### ISSUE TITLE
IllegalStateException in XMPPConnection.login()

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#200]
```
in v3.0

Message ?? 19 juil. 2011 15:38:52 someone keeps breaking in and changing my 
account and passwords help

java.lang.IllegalStateException: Not connected to server.
at org.jivesoftware.smack.XMPPConnection.login(XMPPConnection.java:190)
at org.jivesoftware.smack.Connection.login(Connection.java:357)
at 
com.googlecode.gtalksms.xmpp.XmppAccountManager.makeConnectionAndSavePreferences
(XmppAccountManager.java:103)
at 
com.googlecode.gtalksms.panels.wizard.UseAccountLoginButtonClickListener.onClick
(UseAccountLoginButtonClickListener.java:73)
at android.view.View.performClick(View.java:2408)
at android.view.View$PerformClick.run(View.java:8817)
at android.os.Handler.handleCallback(Handler.java:587)
at android.os.Handler.dispatchMessage(Handler.java:92)
at android.os.Looper.loop(Looper.java:144)
at android.app.ActivityThread.main(ActivityThread.java:4937)
at java.lang.reflect.Method.invokeNative(Native Method)
at java.lang.reflect.Method.invoke(Method.java:521)
at 
com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:868)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:626)
at dalvik.system.NativeStart.main(Native Method)
```

Original issue reported on code.google.com by `Florent....@gmail.com` on 19 Jul 2011 at 3:44

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 19 Jul 2011 at 7:00
- Added labels: **Wizard**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 31 Aug 2011 at 10:59
- Changed title: **IllegalStateException in XMPPConnection.login()**
- Added labels: **FromMarket**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 9 Sep 2011 at 9:46
- Added labels: **Priority-Low**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
i also got the same error
plz help me
```

Original comment by `swati.pi...@gmail.com` on 27 Dec 2011 at 11:16

**Comment by GoogleCodeExporter** [comment]
```
Please provide an app log
```

Original comment by `fschm...@gmail.com` on 27 Dec 2011 at 1:14

**Comment by GoogleCodeExporter** [comment]
```
There is no more wizard with the new UI
```

Original comment by `Florent....@gmail.com` on 23 Jul 2012 at 4:29
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed with GTalkSMS v4.0 and released on market
```

Original comment by `Florent....@gmail.com` on 28 Jul 2012 at 4:20
- Changed state: **Deployed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed with GTalkSMS v4.0 and released on market
```

Original comment by `Florent....@gmail.com` on 28 Jul 2012 at 4:20
